From 4cd12484eae584e42fc92ec8e4088eca0fc0f163 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Mon, 18 Mar 2019 12:59:39 -0700
Subject: [PATCH] TRITON-1317 triton repos.json maint (Mar 2019) Reviewed by:
 Chris Burroughs <chris.burroughs@joyent.com> Approved by: Chris Burroughs
 <chris.burroughs@joyent.com>

---
 repos.json | 17 +++++------------
 1 file changed, 5 insertions(+), 12 deletions(-)

diff --git a/repos.json b/repos.json
index d170399..e75ec78 100644
--- a/repos.json
+++ b/repos.json
@@ -500,16 +500,6 @@
         {
             "name": "sdc-bunyan-serializers"
         },
-        {
-            "name": "sdc-cloud-analytics",
-            "labels": {
-                "release": true,
-                "tritonservice": "ca",
-                "image": "ca",
-                "mg": "ca",
-                "vm": true
-            }
-        },
         {
             "name": "sdc-cloudapi",
             "labels": {
@@ -907,6 +897,9 @@
                 "vm": true
             }
         },
+        {
+            "name": "triton-logarchiver"
+        },
         {
             "name": "triton-mockcloud",
             "labels": {
@@ -979,6 +972,7 @@
         "commons-vfs-manta",
         "conch",
         "conch-docs",
+        "conch-engineering-planning",
         "conch-report-card",
         "conch-reporter-smartos",
         "conch-shell",
@@ -993,7 +987,6 @@
         "cr-update-reviewers",
         "ctf2json",
         "daggr",
-        "dcim_agent",
         "debian-image-tests",
         "debian-lx-brand-image-builder",
         "docker-machine-driver-triton",
@@ -1183,7 +1176,6 @@
         "node-snmpjs",
         "node-splitduplex",
         "node-stackvis",
-        "node-tar-1",
         "node-timefilter",
         "node-wfq",
         "node-zfile",
@@ -1223,6 +1215,7 @@
         "rust-cueball",
         "rust-fast",
         "rust-manta",
+        "rust-muppet",
         "s3-manta-bridge",
         "sdc-hammer",
         "sdc-vmtools",
-- 
2.21.0

